cdap-docs/developers-manual/source/building-blocks/plugins.rst

```diff
@@ -336,7 +336,7 @@ it needs to be set explicitly, as the JAR contents are uploaded without the file
 .. tabbed-parsed-literal::
 
   $ curl -w"\n" -X POST "localhost:10000/v3/namespaces/default/artifacts/mysql-connector-java" \
-  -H "Artifact-Plugins: [ { 'name': 'mysql', 'type': 'jdbc', 'className': 'com.mysql.jdbc.Driver' } ]" \
+  -H 'Artifact-Plugins: [ { "name": "mysql", "type": "jdbc", "className": "com.mysql.jdbc.Driver" } ]' \
```

Comment:
Fixed. And added support for leading whitespace, and fixed a few other examples I noticed.